### PR TITLE
Implement invoice endpoints

### PR DESCRIPTION
**Pull request by kaarelkaarelson**
All invoice endpoints done.

### CODE REVIEW

File: src/API/Invoices.php [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+        return $response;
+    }
+
+    /**
+     * Update the invoice settings of the specified company.
+     *
+     * @see https://rmp-api.rik.ee/api.html#operation/patch-invoice_info
+     *
+     * @param array<string,mixed>|array{
```
Comment (aanndryyyy) [inline src/API/Invoices.php:155]: This new line seems to mess up PHPStan, not passing.

File: src/API/Invoices.php [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                    'term_days',
+                ]
+            ),
+            $parameters
+        );
+
+        if ( count( $missingParameters ) !== 0 ) {
+            $missingKeys = implode( ', ', array_keys( $missingParameters ) );
+
+            throw new \InvalidArgumentException(
```
Comment (aanndryyyy) [inline src/API/Invoices.php:120]: Let's keep using `throw` for errors inside the API Client. Then it is easy to use a `try catch` block when using the Client in PHP solutions.

**Review by aanndryyyy — CHANGES_REQUESTED** [review]
PHPStan pass is the only problem I see at the moment. I will test run the endpoints at some future time.

File: src/API/Invoices.php (hunk 1 above)
Comment (kaarelkaarelson) [reply src/API/Invoices.php:155]: Correct, empty comment line was causing the problem

**Comment by kaarelkaarelson** [comment]
@aanndryyyy Resolved the issue. Please review again.

**Review by aanndryyyy — APPROVED** [review]
All is good!

**Pull request merged**